organization/_posts/2017-09-21-startup.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "HSF Weekly Meeting #109, September 21, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #109, September 21, 2017
+
+#### *Present*: Ian Bird, Michel Jouvin, Dario Menasce, Pere Mato, Liz Sexton-Kennedy, John Harvey, Benedikt Hegner, Marko Petric, Graeme Stewart, Andre Sailer, Daniel Elvira, David Lange, Mike Sokoloff, Pete Elmer, David Lange, Marco Brako, Thomas Kuhr
+
+
+## CWP
+
+-   #### General matters and Editorial Board
+    - See [slides](https://indico.cern.ch/event/667285/contributions/2727738/attachments/1526808/2389640/CWP_Editorial_Board_2017-09-21.pdf)
+    - Editorial Board: trying to address the LHCC comments about the need to involve more people outside the LHC community (despite the fact that LHCC is only concerned with the LHC) and too much CERN+US dominated. Contacts established with Belle II, LC community, IF, ... 
+    - The Reconstruction and Simulation summaries are very good, so we recommend the structure **Challenges + Current Approaches + Roadmap**
+    - It is better to organise R&D actions on 2 time scales: 2y and 5y (rather than 1/3/5). Doesn’t mean that all actions will be delayed by 2 years of course.
+    - Effort description: minimum granularity is whether it is an additional effort to what we are doing or just re-assigning existing effort (e.g., easy things with modest impact should be done)
+    - Synthesis document
+        - A small team set up to prepare a first draft that will be discussed by EB and community
+        - Keep sections 6 and 7 separate for now
+        - An executive summary of the synthesis document is TBD
+
+-   #### Simulation
+    - Got feedback from ATLAS, focused on clarifying the strategy; addressed by Daniel, ATLAS happy with the proposal.
+    - Document now almost finalized: no major change expected
+
+-   #### MachineLearning
+    - No news, Grame will follow up.
+
+-   #### Software Development
+    - Updated document in line with some comments received. Revamping the roadmap section. Should finish tomorrow.
+
+-   #### Software Trigger and Event Reconstruction
+    - No updates, ATLAS were asked to give feedback.
+
+-   #### Data Access, Organisation and Management
+    - Meeting scheduled right after our weekly meeting today: more after it.
```

Comment:
I forgot to update the live notes after our meeting... May be you want to update the status to 
* Contents more or less ok but some work in progress to better define the challenges (currently dispersed in the text) and write a proper R&D Roadmap (currently an unordered, unprioritized bullet list)
* Progress expected at next WG meeting (just after the HSF weekly meeting)